src/js/events.js

```diff
@@ -170,6 +170,17 @@ export class RegularViewEventModel extends RegularVirtualTableViewModel {
         await new Promise(setTimeout);
         const diff = event.pageX - start;
         const override_width = width + diff;
+
+        const {maxWidth, minWidth} = th.dataset;
+        const fixed_max_width = maxWidth && parseInt(maxWidth);
+        const fixed_min_width = minWidth && parseInt(minWidth);
+        const is_oversized = maxWidth && override_width > fixed_max_width;
+        const is_undersized = minWidth && override_width < fixed_min_width;
+
+        // If the column has a data-max-width or a data-min-width attribute,
+        // check that the override width does not go outside these limits.
+        if (is_oversized || is_undersized) return;
```

Comment:
`dataset` is probably not the right way to pass this information, but the underlying issue it exposes is that "column auto sizing" has no API - it is probably a good idea to formalize how column size and automatic size detection are handled first before auto-sizing can be made to clearly not conflict with CSS settings.

Reply:
Thanks! Events file is not longer being affected by this PR.